tests/end_to_end/test_brute_force/test_brute_force_rdp/data/test_cases/cases_brute_force.yaml

```diff
@@ -0,0 +1,11 @@
+- name: rdp_brute_force
+  description: Check if the alert is generated when executing a brute force attack via RDP.
+  configuration_parameters: null
+  metadata:
+    rule.id: 60204
+    rule.level: 10
+    rule.description: Multiple Windows logon failures.
+    extra:
+      mitre_technique: Brute Force
+    extra_vars:
+      agent_os: Windows
```

Comment:
We should no longer need this. Also, please rename the file.

Reply:
Done in 123ef4c40e7ce10616c8601a498cd48decfe62e7